old/server/index.js

```diff
@@ -39,71 +33,71 @@ const REQUESTS = {
  * The result should look like `{"@context":"https://irma.app/ld/request/disclosure/v2","context":"AQ==","nonce":"VItsL+3+GiBHyIt1hIRwSQ==","protocolVersion":"2.5","disclose":[[["pbdf.sidn-pbdf.irma.pseudonym"]]]}`
  */
 
-const createIrmaRequest = content => {
+const createIrmaRequest = (content) => {
+  console.log({ content });
   return {
-    '@context': 'https://irma.app/ld/request/disclosure/v2',
+    "@context": "https://irma.app/ld/request/disclosure/v2",
```

Comment:
Je maakt nu overal " van, je kunt ook de prettier settings in je IDE aanpassen om single quotes te gebruiken?

Reply:
ze zijn weer single